src/index.tsx

```diff
@@ -7,6 +7,8 @@ import { Point } from './types'
 
 type Props = HTMLAttributes<HTMLDivElement> & {
   children: React.ReactNode
+  /** Determines whether drag functionality is enabled, defaults to true */
+  allowDrag?: boolean | undefined
```

Comment:
```suggestion
  allowDrag?: boolean
```

Reply:
Nice catch. Updated that, also updated readme.